File: ServerSpace/src/main/java/Business/EntLista.java
package Business;

import DataLayer.EntListaDB;
import EntityLayer.EntListaEntity;
import java.util.ArrayList;
import Framework.Inj;

public class EntLista {
    public ArrayList<EntListaEntity> ObtenerCodigo(String codigo) {
        try {
            EntListaDB BD = new EntListaDB();
            Inj.IniciarTranssaccionConsulta();
            return BD.ObtenerCodigo(codigo);
        } catch (Exception ex) {
            System.out.println("ERROR " + ex);
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }

    public ArrayList<EntListaEntity> GetListaRelacion_Main(String codigo) {
        try {
            EntListaDB BD = new EntListaDB();
            Inj.IniciarTranssaccionConsulta();
            return BD.GetListaRelacion_Main(codigo);
        } catch (Exception ex) {
            System.out.println("ERROR " + ex);
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }

    public EntListaEntity Registrar(EntListaEntity Item) {
        try {
            Inj.IniciarTranssaccion(true);
            EntListaDB BD = new EntListaDB();
            BD.Registrar(Item);
            Inj.FinalizarTranssaccion();
            return Item;
        } catch (Exception ex) {
            System.out.println("ERROR " + ex);
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }
}

File: SharedApi/Space/Space.BusinessLayer/EntLista.cs
﻿using Framework;
using Space.DataLayer;
using Space.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Space.BusinessLayer
{
    public class EntLista
    {
        public static List<EntListaEntity> ObtenerCodigo(String codigo)
        {
            EntListaDB DB = new EntListaDB();
            return DB.ObtenerCodigo(codigo);
        }

        public static Int32 SaveDB(EntListaEntity Ent)
        {
            EntListaDB DB = new EntListaDB();
            DB.Registrar(Ent);
            return Ent.ListaId;
        }


       

        public static List<EntListaEntity> GetListaRelacion_Main(String codigo)
        {
            EntListaDB DB = new EntListaDB();
            return DB.GetListaRelacion_Main(codigo);
        }
        public static List<EntListaEntity> GetListaRelacionLike(String codigo, String Nombre)
        {
            EntListaDB DB = new EntListaDB();
            return DB.GetListaRelacionLike(codigo, Nombre);
        }



    }
}
